lib/fizzy/types.hpp

```diff
@@ -162,9 +195,63 @@ enum class Instr : uint8_t
     i64_shr_u = 0x88,
     i64_rotl = 0x89,
     i64_rotr = 0x8a,
+
+    f32_abs = 0x8b,
+    f32_neg = 0x8c,
+    f32_ceil = 0x8d,
+    f32_floor = 0x8e,
+    f32_trunc = 0x8f,
+    f32_nearest = 0x90,
+    f32_sqrt = 0x91,
+    f32_add = 0x92,
+    f32_sub = 0x93,
+    f32_mul = 0x94,
+    f32_div = 0x95,
+    f32_min = 0x96,
+    f32_max = 0x97,
+    f32_copysign = 0x98,
+
+    f64_abs = 0x99,
+    f64_neg = 0x9a,
+    f64_ceil = 0x9b,
+    f64_floor = 0x9c,
+    f64_trunc = 0x9d,
+    f64_nearest = 0x9e,
+    f64_sqrt = 0x9f,
+    f64_add = 0xa0,
+    f64_sub = 0xa1,
+    f64_mul = 0xa2,
+    f64_div = 0xa3,
+    f64_min = 0xa4,
+    f64_max = 0xa5,
+    f64_copysign = 0xa6,
+
     i32_wrap_i64 = 0xa7,
+    i32_trunc_f32_s = 0xa8,
+    i32_trunc_f32_u = 0xa9,
+    i32_trunc_f64_s = 0xaa,
+    i32_trunc_f64_u = 0xab,
     i64_extend_i32_s = 0xac,
     i64_extend_i32_u = 0xad,
+    i64_trunc_f32_s = 0xae,
+    i64_trunc_f32_u = 0xaf,
+    i64_trunc_f64_s = 0xb0,
+    i64_trunc_f64_u = 0xb1,
+    f32_convert_i32_s = 0xb2,
+    f32_convert_i32_u = 0xb3,
+    f32_convert_i64_s = 0xb4,
+    f32_convert_i64_u = 0xb5,
+    f32_demote_f64 = 0xb6,
+    f64_convert_i32_s = 0xb7,
+    f64_convert_i32_u = 0xb8,
+    f64_convert_i64_s = 0xb9,
+    f64_convert_i64_u = 0xba,
+    f64_promote_f32 = 0xbb,
+    i32_reinterpret_f32 = 0xbc,
+    i32_reinterpret_f64 = 0xbd,
```

Comment:
```suggestion
    i64_reinterpret_f64 = 0xbd,
```

Reply:
You're correct:

![Screenshot 2020-02-12 at 14 20 25](https://user-images.githubusercontent.com/20340/74343341-d150de00-4da2-11ea-9648-ac70bac6b840.png)

Reply:
Fixed.